website/docs/patterns/disabled-patterns/partials/guidelines/overview.md

```diff
@@ -0,0 +1,8 @@
+We recommend avoiding disabling elements as they are not accessible to users with assistive technology. Disabled elements cannot be navigated to or interacted with, which can cause confusion and frustration. Additionally, our users have given us explicit feedback that they prefer our approach of not displaying disabled elements in the UI.
+
+Instead, we recommend:
+- Show the content/element as enabled and usable.
```

Comment:
Maybe this, or similar?

```suggestion
- Show the content/element as enabled and usable, but provide a half-way point that contextualizes the inability to use this feature.
```

Reply:
Actually, I',m opting to remove that last bit entirely. Because we explain in detail throughout. This should be shorter and to the point because it's an overview.